from app.exceptions.runtime_exception import RuntimeException


class EntityNotFoundException(RuntimeException):
    pass